OPAE/opae-sdk
# Verify and separate events tests to work on hw

* Add missing mock irq functional tests

* fix some formatting

* Add review suggestions

* Re-arrange code in teardown

* Fixed errors

* Increase timeout limit

* Re-enable j4 flag

* Set ctest Timeout for events

* Checkpoint 1: Started seperating mock tests

* Seperate event tests from mock and hw

* Fixed hw tests to work on vm

* Seperate out invalid hw event tests to mock only

* Remove unused vars

* Seperate invalid hw tests to mock only

* Verify all event tests work on hw with mcp and dcp
